lib/tasks/annotate_models.rake

```diff
@@ -21,7 +21,7 @@ task annotate_models: :environment do
   options[:show_indexes] = Annotate.true?(ENV['show_indexes'])
   options[:simple_indexes] = Annotate.true?(ENV['simple_indexes'])
   options[:model_dir] = ENV['model_dir'] ? ENV['model_dir'].split(',') : ['app/models']
-  options[:root_dir] = ENV['root_dir'] ? ENV['root_dir'].split(',') : ['']
+  options[:root_dir] = ENV['root_dir'] && !ENV['root_dir'].empty? ? ENV['root_dir'].split(',') : ['']
```

Comment:
100% yes, we should. :)

I wasn't sure at first of how large the surface of `root_dir` was, so I wanted to start off with this simple fix. It seems though like this could really easily be taken care of inside `AnnotateModels.root_dir`.

Would you like me to give that a shot?